1-exercises/D-array-filter/exercise.js

```diff
@@ -8,7 +8,9 @@
 
 let pairsByIndexRaw = [[0, 3], [1, 2], [2, 1], null, [1], false, "whoops"];
 
-let pairsByIndex; // Complete this statement
+let pairsByIndex = pairsByIndexRaw
+  .filter((item) => item)
+  .filter((item) => item.length === 2); // Complete this statement
```

Comment:
1. Seems innovative in making use of null and false to filter. A more robust way to filter shall rely on positive identification method say **Array.isArray**
2. However, not working well as string "NO" will fit your second filter